starwars/src/components/StarWarsCard.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Card, CardBody, CardText, CardGroup } from "reactstrap";
+
+function StarWarsCard(props) {
```

Comment:
great work! Keep in mind that you can also destructure the object to make your code DRY-er- you could write `const { gender, eye_color, hair_color, name } = props.character` and then skip prepending props.character to each one